snapshot/policy/compression_policy.go

```diff
@@ -0,0 +1,94 @@
+package policy
+
+import (
+	"path/filepath"
+	"sort"
+
+	"github.com/kopia/kopia/fs"
+	"github.com/kopia/kopia/repo/object"
+)
+
+// CompressionPolicy specifies compression policy.
+type CompressionPolicy struct {
+	CompressorName object.CompressorName `json:"compressorName,omitempty"`
+	OnlyCompress   []string              `json:"onlyCompress,omitempty"`
+	NeverCompress  []string              `json:"neverCompress,omitempty"`
+	MinSize        int64                 `json:"minSize,omitempty"`
```

Comment:
Any reason for not making these `uint64`?

Reply:
compression is done on a per-content basis and those are never too big ~O(20MB). If an object is big it will consist of many individually-compressed contents.